editors/vscode/.vscode/launch.json

```diff
@@ -0,0 +1,45 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "extensionHost",
+      "request": "launch",
+      "name": "Launch Client",
+      "runtimeExecutable": "${execPath}",
+      "args": ["--extensionDevelopmentPath=${workspaceRoot}"],
+      "outFiles": ["${workspaceRoot}/out/src/*.js"],
+      "preLaunchTask": {
+	"type": "npm",
+	"script": "watch"
+      }
+    },
+    {
+      "type": "node",
+      "request": "attach",
+      "name": "Attach to Server",
+      "port": 6009,
+      "restart": true,
+      //"outFiles": ["${workspaceRoot}/server/out/**/*.js"]
+      "outFiles": ["${workspaceRoot}/../../lsp-lsp/lp_lsp.ml"]
```

Comment:
Isn't it lp-lsp/ ?

Reply:
Indeed this configuration is not useful; this is only supported for servers in Javascript / Typescript I think [can OCaml be debugged from vscode?].

I will remove it.